test/transcript/DefaultTranscriptionController.test.ts

```diff
@@ -74,12 +72,15 @@ describe('DefaultTranscriptionController', () => {
       transcriptionController.subscribeToTranscriptEvent(callback2);
       realtimeController.realtimeReceiveDataMessage(makeTranscriptDataMessage());
       expect(resultTranscriptEvent1).to.not.be.null;
-      expect(resultTranscriptEvent1.transcript).to.be.undefined;
-      expect(resultTranscriptEvent1.status).to.not.be.null;
-      expect(resultTranscriptEvent1.status.type).to.eq(TranscriptionStatusType.STARTED);
-      expect(resultTranscriptEvent1.status.message).to.eq('message');
-      expect(resultTranscriptEvent1.status.transcriptionRegion).to.eq(`us-east-1`);
-      expect(resultTranscriptEvent1.status.transcriptionConfiguration).to.eq(
+      if (resultTranscriptEvent1 instanceof Transcript) {
+        assert.fail();
+        return;
+      }
+
+      expect(resultTranscriptEvent1.type).to.eq(TranscriptionStatusType.STARTED);
+      expect(resultTranscriptEvent1.message).to.eq('message');
+      expect(resultTranscriptEvent1.transcriptionRegion).to.eq(`us-east-1`);
```

Comment:
For future reference (no need to fix now), convention is to use straight quotes for strings that don't have any substitutions.

Reply:
fixed